modules/lib/core/index.d.ts

```diff
@@ -449,7 +449,7 @@ export interface ErrorController {
 }
 
 export interface IdProviderController extends Controller {
-    autoLogin?: RequestHandler;
+    autoLogin?: void;
```

Comment:
Type cannot be `void`. It should have a special new type where function still accepts `request` but returns `void` instead of `Response`.

Reply:
Yeah, was my initial intent, but got distracted and treated autoLogin as response itself.